content/pt-br/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -0,0 +1,501 @@
+cd /proc/1
+cat status
+```
+
+A saída mostra o bitmap de recursos para o processo:
+
+```
+...
+CapPrm:	00000000a80425fb
+CapEff:	00000000a80425fb
+...
+```
+
+Anote o bitmap dos recursos e depois saia do seu shell:
+
+```shell
+exit
+```
+
+Em seguida, execute um contêiner que seja o mesmo que o contêiner anterior, porém com recursos adicionais definidos.
+
+Aqui está o arquivo de configuração para um Pod que executa um contêiner. A configuração
+adiciona os recursos `CAP_NET_ADMIN` e `CAP_SYS_TIME`:
+
+{{< codenew file="pods/security/security-context-4.yaml" >}}
+
+Crie o Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/pods/security/security-context-4.yaml
+```
+
+Abra um shell no contêiner em execução:
+
+```shell
+kubectl exec -it security-context-demo-4 -- sh
+```
+
+No seu shell, veja os recursos para o processo 1:
+
+```shell
+cd /proc/1
+cat status
+```
+
+A saída mostra os recursos de bitmap para o processo:
+
+```
+...
+CapPrm:	00000000aa0435fb
+CapEff:	00000000aa0435fb
+...
+```
+
+Compare os recursos dos dois contêineres:
+
+```
+00000000a80425fb
+00000000aa0435fb
+```
+
+No bitmap de _capabilities_ do primeiro contêiner, os bits 12 e 25 estão vazios. No segundo contêiner,
+os bits 12 e 25 estão definidos. O bit 12 é `CAP_NET_ADMIN`, e o bit 25 é `CAP_SYS_TIME`.
+Veja [capability.h](https://github.com/torvalds/linux/blob/master/include/uapi/linux/capability.h)
+para definições das constantes de _capabilities_.
+
+{{< note >}}
+As constantes de _capabilities_ do Linux têm o formato `CAP_XXX`.
+Mas quando você listar os recursos no manifesto do seu contêiner, você deve
+omitir a parte `CAP_` da constante.
+Por exemplo, para adicionar `CAP_SYS_TIME`, inclua `SYS_TIME` na sua lista de recursos.
+{{< /note >}}
+
+## Defina o perfil do Seccomp para um Contêiner
+
+Para definir o perfil do Seccomp para um contêiner, inclua o campo `seccompProfile` na seção `securityContext` do seu Pod ou manifesto do contêiner. 
+O campo `seccompProfile` é um objeto
+[SeccompProfile](/docs/reference/generated/kubernetes-api/{{< param "version">}}/#seccompprofile-v1-core) consistindo de `type` e `localhostProfile`.
+As opções válidas para `type` incluem `RuntimeDefault`, `Unconfined`, e
+`Localhost`. O campo `localhostProfile` só deve ser definido quando `type: Localhost`. 
+Este campo indica o caminho do perfil pré-configurado no nó, relativo à localização do perfil de Seccomp do Kubelet 
+(configurado com a flag `--root-dir`).
+
+Aqui está um exemplo que define o perfil Seccomp ao perfil padrão do nó do agente de execução do contêiner:
+
+```yaml
+...
+securityContext:
+  seccompProfile:
+    type: RuntimeDefault
+```
+
+Aqui está um exemplo que define o perfil Seccomp para um arquivo pré-configurado em
+`<kubelet-root-dir>/seccomp/my-profiles/profile-allow.json`:
+
+```yaml
+...
+securityContext:
+  seccompProfile:
+    type: Localhost
+    localhostProfile: my-profiles/profile-allow.json
+```
+
+## Atribua rótulos SELinux a um contêiner
```

Comment:
What do you think about keeping the "label" instead of "rótulo"? On line 433 the example uses "label" and "rótulo" can be confusing.

In other docs the word "label" was kept